src/core/config/child_supervision.h

```diff
@@ -69,7 +57,7 @@
  * The default supervision check timeout interval (in seconds) used by a device in child state. Set to zero to disable
  * the supervision check process on the child.
  *
- * Applicable only if child supervision feature is enabled (i.e., `OPENTHREAD_CONFIG_CHILD_SUPERVISION_ENABLE` is set).
+ * The
```

Comment:
Incomplete?

Reply:
Yes. Added that it can be changed with `otChildSupervisionSetCheckTimeout()` as well.